WebApiCoreSeed.UnitTests/WebApiCoreSeed.UnitTests.csproj

```diff
@@ -5,16 +5,25 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Remove="Resources\**" />

```

Comment:
Thank you @andresmoschini , now that you mention, in the process of coding the tests, at some point I added resource files that were later removed. This seems to be a remanent.